src/commands/general/msfs.ts

```diff
@@ -0,0 +1,20 @@
+import Discord from 'discord.js';
+import { CommandCategories, CommandDefinition } from '../index';
+import { color } from '../../index';
+
+export const msfs: CommandDefinition = {
+    names: ['msfs'],
+    description: "Redirects user to MSFS server for support",
+    category: CommandCategories.GENERAL,
```

Comment:
This command looks like it would fit in a support category rather than general

Reply:
We have no support category currently, would you want me to make one?

Reply:
Yes, you can find a guide on how to make a category in the README

Reply:
Aware of that, will make one pronto.